olp-cpp-sdk-dataservice-write/src/VersionedLayerClient.cpp

```diff
@@ -33,12 +33,12 @@ VersionedLayerClient::VersionedLayerClient(client::HRN catalog,
 VersionedLayerClient::~VersionedLayerClient() = default;
 
 olp::client::CancellableFuture<StartBatchResponse>
-VersionedLayerClient::StartBatch(const model::StartBatchRequest& request) {
+VersionedLayerClient::StartBatch(model::StartBatchRequest request) {
   return impl_->StartBatch(request);
 }
 
 olp::client::CancellationToken VersionedLayerClient::StartBatch(
-    const model::StartBatchRequest& request, StartBatchCallback callback) {
+    model::StartBatchRequest request, StartBatchCallback callback) {
   return impl_->StartBatch(request, std::move(callback));
```

Comment:
```suggestion
  return impl_->StartBatch(std::move(request), std::move(callback));
```

Reply:
VersionedLayerClientImpl watch const ref to request, not sure if needed